ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/functions/[slug]/CancelFunctionButton/CancelFunctionModal.tsx

```diff
@@ -0,0 +1,184 @@
+import { useCallback, useState } from 'react';
+import { Alert } from '@inngest/components/Alert';
+import { NewButton } from '@inngest/components/Button';
+import { RangePicker } from '@inngest/components/DatePicker';
+import { Modal } from '@inngest/components/Modal';
+import { subtractDuration } from '@inngest/components/utils/date';
+
+import Input from '@/components/Forms/Input';
```

Comment:
Could we use the Input from the shared library instead, and add the Label there? (the components in @/components/Forms are marked as deprecated)